### ISSUE TITLE
Breadcrumbs lack visible focus with keyboard navigation

### ISSUE THREAD

**Issue by wenche** [equinor/design-system#1134]
The way Breadcrumbs are implemented with <a> tags without href don't work with keyboard navigation and focus
